### ISSUE TITLE
error

### ISSUE THREAD

**Issue by taniaCHY** [HyeongseokSon1/SRFeat#2]
sample_list = [0,10001,20001,30001,40001,50001,60001,70001,80001]

What does this sentence mean, I am reporting an error here.

**Comment by HyeongseokSon1** [comment]
We made about 100,000 pre-cropped LR-HR patches and used them for training.
That list contains some indices of sample patches for visualization.

You can change this part to read your own sample patches or validation images.

**Comment by taniaCHY** [comment]
If I want to use the DIV2K valid set, how should I modify it?

**Comment by HyeongseokSon1** [comment]
Basically, the current code reads some sample indices from 'train_hr_img_list' (training patch-list) in main_gan_train.py. We do not use any other special validation set for GAN training. We just observe visual changes of some fixed sample iamges.
If you want to other validation images, you should make a new list for validation set, and read images from the list and replace (sample_imgs_384,  sample_imgs_96) by your new images. Of course you should modify some codes such as sizes of placeholder for validation according to the change.

By the way, we uploaded our data augmentation code (using Matlab).
If you use this code for generating dataset, you can use the training code without significant modification.
